README.md

```diff
@@ -143,6 +143,15 @@ if (templateValues.eslint !== 'y') {
 ```
 Note that the string does contain the .ejs suffix, since the ignore applies when reading the file, the string should exactly match the name of the file in your template.
 
+##### `ignoredFolders`
```

Comment:
Any specific reason to use "folders" instead of "directories"?

Reply:
No specific reason. I can change it if needed